website1/src/scss/sections/_main-items-bunker.scss

```diff
@@ -0,0 +1,205 @@
+.main-items-bunker {
+  &-wrapper {
+    width: 100%;
+    background-color: $grey-light;
+  }
+
+  &__item-box {
+    margin: 71px 49px 220px 110px;
+    position: relative;
+  }
+
+  &__item {
+    position: absolute;
+
+    &-title {
+      text-transform: uppercase;
+      font-size: 24px;
+      line-height: 30px;
+      margin-left: 45px;
+    }
+
+    &-text {
+      font-weight: 300;
+      line-height: 22px;
+      margin-top: 5px;
+      margin-left: 45px;
+
+      &-overlap {
+        max-width: 326px;
+      }
+
+      &-door {
+        max-width: 475px;
+      }
+
+      &-hatch {
+        max-width: 360px;
+      }
+
+      &-ventilation {
+        max-width: 306px;
+      }
+    }
+
+    &-pointer-overlap {
+      position: absolute;
+      top: 81px;
+    }
+
+    &-door {
+      bottom: -120px;
+    }
+
+    &-pointer-door {
+      position: absolute;
+      bottom: 125px;
+    }
+
+    &-hatch {
+      margin-left: 698px;
+    }
+
+    &-pointer-hatch {
+      position: absolute;
+      right: -13px;
+      top: 81px;
+    }
+
+    &-ventilation {
+      bottom: -76px;
+      margin-left: 615px;
+    }
+
+    &-pointer-ventilation {
+      position: absolute;
+      right: 133px;
+      bottom: 86px;
+    }
```

Comment:
For all `pointers` you can create a general class for set `position: absolute`
And better change abstraction, rewrite classes as `door-pointer`, `hatch-pointer` etc. because of pointer less weight. In the future, this will help you manage the display of all pointers.